public/locales/az/gamepage.json

```diff
@@ -17,6 +17,15 @@
             "message": "Bu oyunu təmir etməyə cəhd etmək istəyirsiniz? Bu uzun müddət çəkə bilər.",
             "title": "Doğrulayın və Təmir edin"
         },
+        "runexe": {
+            "title": ""
```

Comment:
this empty string might create issues, I imagine it's missing a default value somewhere

Reply:
fixed